learn/react-native/app-loader.md

```diff
@@ -0,0 +1,101 @@
+---
+title: "Application Loader"
+id: "app-loader"
+sidebar_label: "App Loader"
+---
+---
+
+The Application loader shows animation while variables are loading on a page. 
+
+Initially, it will show the animation for Startup Variables then it will show the animation for the Page-level Variables spinner context option. 
```

Comment:
In WaveMaker service calls to fetch data are abstracted as Variables (provide link to reference Variables page)